<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
background-color: rgb(19, 32, 49);
}

QPushButton {
  text-decoration: none;
  text-transform: uppercase;
  letter-spacing: 2px;
  color: black;
  background: rgb(1, 97, 113);
  font-size: 10px;
  text-align: center;
  border-radius: 10px;
  border: 1px solid rgb(36, 172, 194);
 }

QPushButton:hover {
    background-color: rgb(36, 172, 194);
  }

QLabel {
	color: white;
}

QProgressBar{
border-radius: 5px;
text-align:center;
color: #4f4f4f;
}

QProgressBar::chunk{
border-radius: 5px;
background-color: rgb(3, 227, 255);
}

QTextEdit{
border:1px solid white;
border-radius: 10px;
margin:4px;
color:white;
}

QTextEdit:focus{
border: 1px solid  rgb(36, 172, 194);
}

QSlider::groove{
    border: 1px solid white;
	border-radius: 5px;
    height: 8px; 
    background: qlineargradient(x1:0, y1:0, x2:0.7, y2:0, stop:0  rgb(19, 32, 49), stop:1  rgb(3, 227, 255));
    margin: 2px 1px;
}

QSlider::handle{
    background:white;
    border: 1px solid #5c5c5c;
    width: 18px;
    margin: -2px; 
    border-radius: 3px;
}
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>       Громкость динамиков</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="speaker">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar_2">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>85</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите ссылку на сайт</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="url">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>191</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>ОК</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>45</y>
     <width>561</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Для создания своей собственной команды заполните следующие поля</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="command">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>611</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semilight</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Введите голосовую команду, которую будет обрабатывать Пирс</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
background-color: rgb(11, 23, 35);
}

QLabel {
color: white;
}

QPushButton{
border: 0px;
}

</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="controlButtons" stretch="0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="Roll_up">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/commandButton2.ico</normaloff>icons/commandButton2.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Exit">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/controlButton.ico</normaloff>icons/controlButton.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>31</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="logo">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="logo_ico">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>icons/Settings.ico</normaloff>icons/Settings.ico</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="logo_name">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="titleBtn">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>0</y>
      <width>570</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>272</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Для контакта с разработчиками, </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>     в целях предложения улучшений </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>в рамках проекта пишите нам на почту </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>Yu Gothic UI Semilight</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>             p1rsmakpet@gmail.com</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
